README.md

```diff
@@ -23,64 +23,46 @@ OFRAK equips users with:
 
 <div align="center">
-<img id="ofrak-animation" src="docs/assets/ofrak_gui_1.png" style="max-width:1000px;width:100%">
+<img src="docs/assets/ofrak_gui_1.png">
 </div>
 
-
 ## Getting Started
-See [INSTALL.md](INSTALL.md) for instructions on how to install OFRAK.
 
 **OFRAK uses Git LFS. This means that you must have Git LFS installed before you clone the repository!** Install Git LFS by following [the instructions here](https://git-lfs.github.com/). If you accidentally cloned the repository before installing Git LFS, `cd` into the repository and run `git lfs pull`.
 
-## License
-The code in this repository comes with an [OFRAK Community License](./LICENSE), which is intended for educational uses, personal development, or just having fun.
-
-Users interested in using OFRAK for commercial purposes can request the Pro License, which for a limited period is available for a free 6-month trial. See [OFRAK Licensing](https://ofrak.com/license/) for more information.
+See [`docs/environment-setup`](https://ofrak.com/docs/environment-setup.html) for detailed instructions on how to install OFRAK.
 
 ## Documentation
-OFRAK has general documentation and API documentation. They can be viewed at <https://ofrak.com/docs>.
 
-If you wish to make changes to the documentation or serve it yourself, once OFRAK has been installed, run `make develop` in each of `ofrak_type`, `ofrak_io`, `ofrak_core`, and `ofrak_patch_maker` to install the dependencies needed to work with this documentation source.
+OFRAK has general documentation and API documentation. Both can be viewed at [ofrak.com/docs](https://ofrak.com/docs).
 
-The source for OFRAK's documentation resides in `./docs` and is built using [MkDocs](https://www.mkdocs.org/).
+If you wish to make changes to the documentation or serve it yourself, follow the directions in [`docs/README.md`](docs/README.md).
 
-`mkdocs.yml` contains the configuration information for that source.
+## License
 
-To host and view documents locally, run the following command from the root directory of this repository:
-```
-% mkdocs serve
-INFO     -  Building documentation...
-INFO     -  Cleaning site directory
-INFO     -  Documentation built in 0.14 seconds
-INFO     -  [23:59:36] Serving on http://127.0.0.1:8000/```
-```
+The code in this repository comes with an [OFRAK Community License](LICENSE), which is intended for educational uses, personal development, or just having fun.
 
-Note that the docs must be built (or served using `mkdocs serve`) in an environment with all Python dependencies installed.
+Users interested in OFRAK for commercial purposes can request the Pro License, which for a limited period is available for a free 6-month trial. See [OFRAK Licensing](https://ofrak.com/license/) for more information.
 
 ## Contributing
```

Comment:
Now that you have moved the contributor's guide to `CONTRIBUTING.md`, I think we can remove this duplicated text and just have this section reference `CONTRIBUTING.md`?